src/components/postgresConnect.ts

```diff
@@ -472,7 +472,7 @@ class PostgresDB {
 
   public getCurrentStageHistory = async (applicationId: number) => {
     const text =
-      'SELECT stage_id, stage_number, stage, stage_history_id, status_history_id, status FROM application_stage_status_all WHERE application_id = $1 AND stage_is_current = true'
+      'SELECT stage_id, stage_number, stage, stage_history_id, status_history_id, status FROM application_stage_status_all WHERE application_id = $1 AND stage_is_current = true AND status_is_current = true'
```

Comment:
Best to just use the `application_stage_status_latest` View then you don't need the `current=true`.

Reply:
I'll make changes on new PR